src/components/Layout/Layout.tsx

```diff
@@ -28,8 +28,10 @@ import { Menu } from '@/components/Menu';
 import { LayoutProvider } from '@/components/Layout';
 import { PlatformNavigator } from '@/components/PlatformNavigator';
 import directory from 'src/directory/directory.json';
+import flatDirectory from 'src/directory/flatDirectory.json';
```

Comment:
I'm importing this in too, but hopefully we can come back to in the future. The ideal situation would be that we get this inside getServerSideProps and pass it down as props to be used for each page